docs/assets/sass/docs.scss

```diff
@@ -0,0 +1,53 @@
+@import "govuk-elements";
+
+.markdown {
+	h1 {
+		@extend .heading-xlarge;
+	}
+
+	h2 {
+		@extend .heading-large;
+	}
+
+	h3 {
+		@extend .heading-medium;
+	}
+
+	h4 {
+		@extend .heading-small;
+	}
+
+	code {
```

Comment:
can't remember who wrote this originally, but why isnt the code block just @extend .code?